contracts/utils/Sweeper.sol

```diff
@@ -25,17 +25,14 @@ contract Sweeper is Ownable {
         IERC721 nftaddress;
         uint256[] ids;
     }
-    mapping (address => bool) public lockedTokens;
+    mapping(address => bool) public lockedTokens;
     bool public allowNativeSweep;
 
     event SweepWithdrawToken(address indexed receiver, IERC20 indexed token, uint256 balance);
 
     event SweepWithdrawNFTs(address indexed receiver, NFT[] indexed nfts);
 
-    event SweepWithdrawNative(
-        address indexed receiver,
-        uint256 balance
-    );
+    event SweepWithdrawNative(address indexed receiver, uint256 balance);
```

Comment:
This was probably done by husky, which runs a local pipeline (linting,testing) before any commit.